projects/demo-playwright/tests/addon-charts/line-chart/line-chart.spec.ts

```diff
@@ -0,0 +1,15 @@
+import {TuiDocumentationPagePO, tuiGoto} from '@demo-playwright/utils';
+import {expect, test} from '@playwright/test';
+
+test.describe(`LineChart`, () => {
+    test(`should not show hint`, async ({page}) => {
+        await tuiGoto(page, `charts/line-chart`);
+        const example = new TuiDocumentationPagePO(page).getExample(`#line`);
+        const lineChart = example
+            .locator(`tui-line-chart-example-1 tui-line-chart .t-column`)
+            .first();
+
+        await lineChart.hover();
+        await expect(lineChart).toHaveScreenshot(`01-line-chart.png`);
```

Comment:
It generates the following screenshot:
<img height="200" src="https://github.com/taiga-family/taiga-ui/assets/35179038/705ba47e-6a3a-46b9-b787-8e7af9fd4640" >

Please, change screenshot area
```suggestion
        await expect(example).toHaveScreenshot(`01-line-chart.png`);
```

Reply:
```ts
await tuiGoto(page, `charts/line-chart`);
const example = new TuiDocumentationPagePO(page).getExample(`#line`);

await example
    .locator(`tui-line-chart-example-1 tui-line-chart .t-column`)
    .first()
    .hover();

await expect(example).toHaveScreenshot(`01-line-chart.png`);
```